src/components/button/Button.tsx

```diff
@@ -0,0 +1,37 @@
+import { Button as HDSButton } from 'hds-react';
+import React, { ReactNode } from 'react';
+import './Button.scss';
+
+type ButtonVariant = 'primary' | 'secondary';
+type ButtonTypeVariant = 'button' | 'submit' | 'reset' | undefined;
+
+interface ButtonProps {
+  children: ReactNode;
+  onClick?: () => void;
+  variant?: ButtonVariant;
+  dataTest?: string;
+  className?: string;
+  type?: ButtonTypeVariant;
+}
+
+export default function Button({
+  variant = 'primary',
```

Comment:
Should we make separate components for _PrimaryButton_ and _SecondaryButton_? Or do you think applying the variant prop is enough? I think we might need to make a third kind of button also, so this might not scale very well in the long run. But maybe we think about that when the time comes?

Reply:
It might be better to split those two. I was already thinking icon-button etc. 😁

Reply:
- [ ] Create separate components for _PrimaryButton_ and _SecondaryButton_